src/app/app.component.ts

```diff
@@ -26,10 +26,11 @@ import { globals } from './globals';
 export class AppComponent {
 
   readonly featureToggles = globals.featureToggles;
-
   loginButton = 'Login';
   welcomeText = '';
   user: PAUser;
+  patternLanguagesPath: string = globals.pathConstants.patternLanguages;
+  patternViewsPath: string = globals.pathConstants.patternViews;
```

Comment:
May consider using something like `readonly pathConstants = globals.pathConstants;` here and `pathConstants.patternLanguagesPath` (and similar) in the template. As this can be simpler reused with the other URLs used in the template.